aviatrix/resource_aviatrix_firenet.go

```diff
@@ -266,14 +267,18 @@ func resourceAviatrixFireNetRead(d *schema.ResourceData, meta interface{}) error
 		fI["attached"] = instance.Enabled == true
 		if instance.VendorType == "Aviatrix FQDN Gateway" {
 			fI["vendor_type"] = "fqdn_gateway"
+			if instance.LanInterface[0:4] == "eni-" {
```

Comment:
This is a little unsafe. You will crash the program if the length of `instance.LanInterface` is not at least 4. The safer option is to use `strings.HasPrefix`

Reply:
fixed